packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -212,6 +219,7 @@ function ShoppingPane(props) {
             clearId="clear-product-search"
             handleClearInput={handleClearInput}
           />
+          <ProductSort onChange={onSortBy} sortId={`${orderby}-${order}`} />
```

Comment:
Not related to this specific line but let's expand the existing Karma tests to also cover product sorting

Reply:
Added a test here - https://github.com/GoogleForCreators/web-stories-wp/pull/11552/commits/8008da617a5586b831239e437fdb4b1217639656#diff-fc3f39bc6c4a8d0d2df87c0807cb1ac0c536bc846029b6a67c42a2e9ba74f84bR125